frontend/src/components/profile/components/user-info/user-info.tsx

```diff
@@ -52,6 +55,21 @@ const UserInfo: React.FC<Props> = ({ user, onEdit }) => {
           </div>
         </div>
       </div>
+      {isDoctor && (
+        <Documents
+          document={
+            (user as IUserTypeDoctor).doctor
+              ? (user as IUserTypeDoctor).doctor.document
+              : {
+                id: 'nerwksfd',
+                imagePath: '',
+                status: DocumentStatus.IN_REVIEW,
+                createdAt: '',
+                updatedAt: '',
+              }
+          }
+        />
+      )}
```

Comment:
```suggestion
      {user.doctor.document && (
        <Documents
          document={document}
        />
      )}
```

Reply:
oops, forgot to delete -_-

Reply:
Merge with dev. It's already fixed there